cmd/bundle/run.go

```diff
@@ -19,7 +19,27 @@ func newRunCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run [flags] KEY",
 		Short: "Run a resource (e.g. a job or a pipeline)",
-		Args:  root.MaximumNArgs(1),
+		Long: `Run a resource (e.g. a job or a pipeline)
+
+The KEY argument is the unique identifier of the resource to run. You can customize the run by
+specifying any of the flags below, or if applicable, additional positional arguments.
+
+Additional positional arguments can be specified like this:
+
+   databricks bundle run my_job -- --key1 value1 --key2 value2
+
+Or:
+
+   databricks bundle run my_job -- example positional arguments
+
+If the specified job uses job parameters, the former example applies and flag names are mapped
+to the job parameter names.
```

Comment:
```suggestion
If the specified job uses job parameters or if the job has a notebook task with parameters, the former example applies and flag names are mapped to the parameter names.
```

Reply:
The original is really fine too, but it is easier for me personally to comprehend grouping conditions for when each example applies.

Reply:
Another thought: "first example" instead of "former example"?